radio/src/gui/128x64/view_statistics.cpp

```diff
@@ -41,15 +46,38 @@ void menuStatisticsView(event_t event)
       break;
   }
 
+#if defined(PCBTARANIS)
```

Comment:
Just checked. TIMERS defaults to 3 on all ARM platforms. You can set timers to 2 by manually compiling but that probably also breaks in Taranis/Horus world. (We might just remove that TIMER=2 option).

The only platform which is a bit strange is sky9x with reva board. There the timer might just always be zero. But I think showing a zero timer isn't problematic there. 

I would rather have all arm platforms the same here.